components/workflow-mgt/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/dao/BPSProfileDAO.java

```diff
@@ -200,18 +195,20 @@ public BPSProfile getBPSProfile(String profileName, int tenantId, boolean isWith
         String decryptPassword;
         CryptoUtil cryptoUtil = CryptoUtil.getDefaultCryptoUtil();
         try {
-            Object classCheckresult = null;
+            Object classCheckResult = null;
+            //Checks if IS has BPS features installed with it
             try {
-                classCheckresult = Class.forName("org.wso2.carbon.humantask.deployer.HumanTaskDeployer");
+                classCheckResult = Class.forName("org.wso2.carbon.humantask.deployer.HumanTaskDeployer");
             } catch (ClassNotFoundException e) {
-
+                //If BPS features are not installed, it will throw a ClassNotFoundException, no actionto be executed
```

Comment:
this comment doesn't explain actual reasoning behind 'not doing anything'
may be a more descriptive comment makes sense?